### ISSUE TITLE
Training has very high errors

### ISSUE THREAD

**Issue by hughrawlinson** [hughrawlinson/web-audio-feature-based-synthesis#1]
Following my last commit, ea4af78, I grabbed the output (open the webpage in a browser and `console.log(JSON.stringify(data));` once it finishes whirring) and normalized it. I started to try to train the neural network using [brain.js](https://github.com/harthur/brain). I did this in a node repl, so...

```
var brain = require('brain');
var nn = new brain.NeuralNetwork();
nn.train([...data from browser...]);
```

...but it always maxes out its number of iterations and maintains quite a high error level. I've messed around a bit with the NN settings, like the number of hidden layers, the learning rate, but haven't been able to train to a sub-threshold error at all, never mind reliably. I started stripping out synth parameters that I didn't think would have a big effect on the timbre of the sound, like the final gain, and then started to try and train solely the Spectral Centroid, still couldn't get it to converge.

Context: This is an attempt to train a model for feature based audio synthesis in the browser. The 'data' that I'm talking about is a set of features (namely in the current code, RMS and Spectral Centroid), and corresponding values for parameters on a simple two-oscillator-and-a-low-pass synth. The javascript in this repo generates a set of 10 random synth settings and their corresponding audio features over a second of signal generation (though with a synth like this I would expect timbral features to remain fairly static over the course of a second) using the OfflineAudioContext. (I tried the MOAR DATA approach by generating a set of 100 data points, but Firefox ended up using over 65GB of VIRT memory somehow, so there was probably, hopefully, a bug in my js).

Any thoughts would be greatly appreciated!
